### ISSUE TITLE
Result Comparsion

### ISSUE THREAD

**Issue by Kimsure** [destiny301/SSR#3]
Did other SR methods (SwinIR, HAT) also trained on the same dataset ? Or just use the pretrained model ?

**Comment by destiny301** [comment]
yes, the baselines were trained (strictly speaking, fine-tuned) on the same dataset. Otherwise, the performance is very bad.
